bus/client.go

```diff
@@ -491,6 +494,32 @@ func (c *Client) GougingParams() (gp api.GougingParams, err error) {
 	return
 }
 
+// Accounts returns the ephemeral accounts for a given owner.
+func (c *Client) Accounts(owner string) (accounts []ephemeralaccounts.Account, err error) {
+	err = c.c.GET(fmt.Sprintf("/accounts/%s", api.ParamString(owner)), &accounts)
+	return
+}
+
+// AddBalance adds the given amount to an account's balance.
+func (c *Client) AddBalance(id rhpv3.Account, owner string, hk types.PublicKey, amt *big.Int) (err error) {
+	err = c.c.POST(fmt.Sprintf("/accounts/%s/add", id), api.AccountsAddBalanceRequest{
+		Host:   hk,
+		Owner:  owner,
+		Amount: amt,
+	}, nil)
+	return
+}
+
+// UpdateBalance sets the given account's balance to a certain amount.
+func (c *Client) UpdateBalance(id rhpv3.Account, owner string, hk types.PublicKey, amt *big.Int) (err error) {
```

Comment:
`nit` I think `SetBalance` would be slightly more consistent. For me `UpdateBalance` almost feels like it could be doing the same thing an `AddBalance` would, especially since we're dealing with `big.Int` now that can be negative.